Tests/CLITests/Subcommands/Networks/TestCLINetwork.swift

```diff
@@ -0,0 +1,100 @@
+//===----------------------------------------------------------------------===//
+// Copyright © 2025 Apple Inc. and the container project authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//===----------------------------------------------------------------------===//
+
+//
+
+import AsyncHTTPClient
+import ContainerClient
+import ContainerizationExtras
+import ContainerizationOS
+import Foundation
+import Testing
+
+class TestCLINetwork: CLITest {
+    private static let retries = 10
+    private static let retryDelaySeconds = Int64(3)
+
+    @available(macOS 26, *)
+    @Test func testNetworkCreateAndUse() async throws {
+        defer {
+
+        }
```

Comment:
Was this meant to be deleted?

Reply:
It should be deleted, but it prompted me to look over the test once more and I think what might have happened here was that I had some network cleanup code in a defer, but also I needed to delete the network at the start of the test to ensure a known initial condition.

I think it'd be good to do a belt and suspenders for the network cleanup, so I'm testing a change where I defer delete the network at the end of the test, and also delete the network at the start.